chia/full_node/full_node.py

```diff
@@ -948,22 +950,26 @@ async def sync_from_fork_point(
         target_peak_sb_height: uint32,
         peak_hash: bytes32,
         summaries: List[SubEpochSummary],
-    ):
+    ) -> None:
         buffer_size = 4
         self.log.info(f"Start syncing from fork point at {fork_point_height} up to {target_peak_sb_height}")
-        peers_with_peak = self.get_peers_with_peak(peak_hash)
+        peers_with_peak: List[ws.WSChiaConnection] = self.get_peers_with_peak(peak_hash)
         fork_point_height = await check_fork_next_block(
             self.blockchain, fork_point_height, peers_with_peak, node_next_block_check
         )
         batch_size = self.constants.MAX_BLOCK_COUNT_PER_REQUESTS
 
-        async def fetch_block_batches(batch_queue: asyncio.Queue, peers_with_peak: List[ws.WSChiaConnection]):
+        async def fetch_block_batches(
+            batch_queue: asyncio.Queue[Optional[Tuple[ws.WSChiaConnection, List[FullBlock]]]]
+        ) -> None:
+            start_height, end_height = 0, 0
```

Comment:
Note that this creates a tuple first then unpacks it into the variables.  Probably irrelevant overhead here, but I'm not sure it enhances the code much either.  The form `x = y = 27` could also be used, but making a habit of that hazards doing stuff like `x = y = []` and having just one list created and shared by both variables.

Reply:
I prefer to have less lines, the overhead is zero in this case

Reply:
Although thanks for teaching me that, I did not know it.